content/learn/quick-start/getting-started/setup.md

```diff
@@ -202,7 +223,7 @@ Bevy can be built just fine using default configuration on stable Rust. However
 
 * **Generic Sharing**: Allows crates to share monomorphized generic code instead of duplicating it. In some cases this allows us to "precompile" generic code so it doesn't affect iterative compiles. This is only available on nightly Rust.
 
-To enable fast compiles, install the nightly rust compiler and LLD. Then copy the contents of [this file](https://github.com/bevyengine/bevy/blob/main/.cargo/config_fast_builds.toml) to `YOUR_WORKSPACE/.cargo/config.toml`. For the project in this guide, that would be `my_bevy_game/.cargo/config.toml`.
+To enable fast compiles, install the nightly rust compiler and LLD. Then copy the contents of [this file](https://github.com/bevyengine/bevy/blob/main/.cargo/config_fast_builds.toml) to `<YOUR_WORKSPACE>/.cargo/config.toml`. For the project in this guide, that would be `my_bevy_game/.cargo/config.toml`.
```

Comment:
Should we point at the `latest` ref, maybe? unlikely to matter in practice

Reply:
Agreed, the website should probably prefer using `latest` unless mentioning a specific commit or unstable feature.